src/components/Send.jsx

```diff
@@ -491,20 +499,31 @@ export default function Send() {
 
   return (
     <>
-      <div className="send">
+      <div
+        className={`send ${serviceInfo?.makerRunning && 'maker-running'} ${
+          serviceInfo?.coinjoinInProgress && 'taker-running'
+        }`}
+      >
         <PageTitle title={t('send.title')} subtitle={t('send.subtitle')} />
-
-        <rb.Fade in={!isCoinjoinOptionEnabled} mountOnEnter={true} unmountOnExit={true}>
-          <>
-            {serviceInfo?.makerRunning || serviceInfo?.coinjoinInProgress ? (
-              <div className="mb-4 p-3 border border-1 rounded">
-                <small className="text-secondary">
-                  {serviceInfo?.makerRunning && t('send.text_maker_running')}
-                  {serviceInfo?.coinjoinInProgress && t('send.text_coinjoin_already_running')}
-                </small>
-              </div>
-            ) : null}
-          </>
+        <rb.Fade in={isOperationDisabled()} mountOnEnter={true} unmountOnExit={true}>
+          <rb.Alert variant="info" className="mb-4">
+            <>
+              {serviceInfo?.makerRunning && (
+                <rb.Row>
```

Comment:
```suggestion
                <rb.Row className="align-items-center">
```

To center the caret vertically:

<img width="693" alt="Screenshot 2022-04-20 at 16 04 37" src="https://user-images.githubusercontent.com/10026790/164248615-3932aeff-60c7-4f49-87d1-f2f165d3eeed.png">

Reply:
Done!